config/locales/en.yml

```diff
@@ -916,6 +916,8 @@ en:
       events: "announcements of events"
       reports: "reports from a field test or meetup"
       video: "Watch a 30 second video on posting research notes &raquo;"
+      rich:
+      drag_image: "Drag an image here to upload."
```

Comment:
Great! Thanks @Tilakraj0308, this is really close.

The indentation here matters, because the file is in a YAML format which you can read about [here](https://en.wikipedia.org/wiki/YAML).

Here's what this PR currently has:
```
      events: "announcements of events"
      reports: "reports from a field test or meetup"
      video: "Watch a 30 second video on posting research notes &raquo;"
      rich:
      drag_image: "Drag an image here to upload."
```

But what we're going for is instead:
```
      events: "announcements of events"
      reports: "reports from a field test or meetup"
      video: "Watch a 30 second video on posting research notes &raquo;"
    rich:
      drag_image: "Drag an image here to upload."
```

Can you make another commit to make this change?